add_even_numbers_my_code.py

```diff
@@ -0,0 +1,10 @@
+target = int(input()) # Enter a number between 0 and 1000
+# 🚨 Do not change the code above ☝️
+
+# Write your code here 👇
+total = 0
+for number in range(1,target+1):
+    if(number%2 == 0):
+        total+=number
```

Comment:
Hi, pls try to use inbuilt functions like step parameter in range, shown in below code. It enhances the code. Thank you

Reply:
sure, thank you